extension/src/extension.ts

```diff
@@ -76,6 +66,14 @@ export async function activate(context: ExtensionContext) {
         // onDidChange.fire();
     });
 
+    workspace.onDidCloseTextDocument((document) => {
+        const uri = document.uri;
+        // if a test file is removed, should update test storage.
+        if (!fs.existsSync(uri.fsPath)) {
+            testResourceManager.storeTests(uri, undefined);
```

Comment:
remove tests in the method named `storeTests` is confusing. Can we make another method named something like `removeTests()`?

Reply:
make sense